### ISSUE TITLE
Newegg Canada Item added to cart but still says "Card not in Stock"

### ISSUE THREAD

**Issue by MrHarBear** [Ataraksia/NeweggBot#9]
First of all really appreciate you building this tool!

I am trying to use this on Newegg Canada to get an RX 6000s card this Wednesday. I followed the instruction and downloaded the Canada branch. I used a dummy item and the script runs fine adding the item into cart but it will not check out. Please see the screenshot for reference.

![image](https://user-images.githubusercontent.com/42833195/99302189-f0efc800-2846-11eb-97c0-130e7ee0f679.png)

**Comment by Codolophier** [comment]
This could be due to a change by Newegg in the HTML element for the cart total.  Check [this issue thread](https://github.com/Ataraksia/NeweggBot/issues/12) for a possible solution.  

TL;DR:  Try changing the first part of the **check_cart** method from:

```
try {
  ...
  var element = await page.$('span.amount')
  var text = await page.evaluate(element => element.textContent, element);
 ...
 }
...
```
to:

```
const amountElementName = ".summary-content-total"
 try {
   await page.waitForSelector(amountElementName, { timeout: 1000 })
   var amountElement = await page.$(amountElementName)
   var text = await page.evaluate(element => element.textContent, amountElement)
    ....
    }
   ....
```
